numalogic/udfs/inference.py

```diff
@@ -95,6 +103,10 @@ def exec(self, keys: list[str], datum: Datum) -> Messages:
         # Construct payload object
         payload = StreamPayload(**orjson.loads(datum.value))
 
+        MSG_IN_COUNTER.increment_counter(
+            self.__class__.__name__, ":".join(payload.composite_keys), payload.config_id
```

Comment:
Let's add a `_vtx = "inference"` or something of that sort as a class variable.

Reply:
Sure. Let me introduce this in base class itself